packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats.scss

```diff
@@ -30,24 +30,26 @@
 }
 
 .accumulators-stats {
-    background: var(--general-main-1);
+    background: var(--semantic-color-slate-solid-surface-frame-low);
     padding: var(--core-spacing-400);
     margin-top: calc(var(--core-spacing-600) * -1);
+    border-bottom-left-radius: var(--core-borderRadius-400);
+    border-bottom-right-radius: var(--core-borderRadius-400);
     &__container {
         display: flex;
         justify-content: space-between;
         &__heading {
             margin: var(--core-spacing-300) var(--core-spacing-400);
             background: transparent;
             border: none;
-            border-bottom: var(--core-spacing-75) dashed;
+            border-bottom: var(--core-spacing-75) dashed var(--component-textIcon-normal-default);
             padding: var(--core-spacing-50);
         }
         &__divider {
-            border-right: var(--core-spacing-75) solid var(--core-color-opacity-black-100);
+            border-right: var(--core-spacing-75) solid var(--semantic-color-monochrome-surface-normal-low);
             margin: var(--core-spacing-50) var(--core-spacing-400);
         }
-        .slide-right {
+        .-slide-right {
```

Comment:
Is `-` expected here? Looks unusual, when `class` started from `-`

Reply:
yeah its a typo